.github/workflows/build.yml

```
@@ -0,0 +1,76 @@
+      - uses: ./.github/actions/setup-nix
+      - name: Find x86 Jobs
+        id: x86_64
+        run: echo "packages=$(nix eval .#hydraJobs.packages.x86_64-linux --apply "builtins.attrNames" --json)" >> "$GITHUB_OUTPUT"
+      - name: Find aarch64 Jobs
+        id: aarch64
+        run: echo "packages=$(nix eval .#hydraJobs.packages.aarch64-linux --apply "builtins.attrNames" --json)" >> "$GITHUB_OUTPUT"
+      - name: Find test Jobs
+        id: tests
+        run: echo "tests=$(nix eval .#hydraJobs.packages.tests --apply "builtins.attrNames" --json)" >> "$GITHUB_OUTPUT"
+  build-x86_64:
+    runs-on: blacksmith-8vcpu-ubuntu-2404
```

actionlint (1.7.11)

[error] label "blacksmith-8vcpu-ubuntu-2404" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)